python/llm/example/CPU/Deepspeed-AutoTP/run_deepspeed.sh

```diff
@@ -0,0 +1,20 @@
+#/bin/bash
+export WORLD_SIZE=2
+export REPO_ID_OR_MODEL_PATH=/home/llm/models/Llama-2-7b-chat-hf
+export OMP_NUM_THREADS=48
+
+export MASTER_ADDR=127.0.0.1
+export CCL_ZE_IPC_EXCHANGE=sockets
+export CCL_ROOT=/opt/intel/oneccl
```

Comment:
Can we remove oneCCL related path? It's installation path related.

Reply:
> Can we remove oneCCL related path? It's installation path related.

It's OK for now; let's focus on getting optimized performance first